schema/donation-summary-no-orm.sql

```diff
@@ -0,0 +1,8 @@
+-- Auto generated file, do not edit.
+-- run ./matchbot matchbot:write-schema-files to update
+
+-- Note that CHARSET and COLLATE details have been removed to allow sucessful comparisons between schemas
+-- on dev machines and CircleCI. Do not use these files to create a database. They are provided for
+-- information only.
+
+CREATE ALGORITHM=UNDEFINED DEFINER=`root`@`%` SQL SECURITY DEFINER VIEW `donation-summary-no-orm` AS select `Donation`.`id` AS `id`,`Donation`.`salesforceId` AS `salesforceId`,`Donation`.`createdAt` AS `createdAt`,`Donation`.`amount` AS `amount`,`Donation`.`donationStatus` AS `donationStatus`,`Charity`.`name` AS `CharityName`,`Campaign`.`name` AS `CampaignName` from ((`Donation` join `Campaign` on((`Campaign`.`id` = `Donation`.`campaign_id`))) join `Charity` on((`Charity`.`id` = `Campaign`.`charity_id`)))
```

Comment:
`no-orm` in the name was intended to allow us to have doctrine filter it out when checking for migrations etc, but it turns that's not necessary - the schema introspection ignores everything other than 'base tables', i.e. ignores views by default. See implementation: https://github.com/doctrine/dbal/blob/3.9.x/src/Platforms/AbstractMySQLPlatform.php#L384

If we did need to filter stuff ourselves we can do it by passing a predicate that takes a table name to `\Doctrine\DBAL\Configuration::setSchemaAssetsFilter`, or by using `--filter-expression=FILTER-EXPRESSION` CLI argument for vendor/bin/doctrine-migrations migrations:diff

Reply:
slightly annoying that mysql seems to always output view create table statements one line like this, but not that worthwhile to fix.